doc/CHANGELOG.md

```diff
@@ -8,7 +8,7 @@ Syntax: `- short text describing the change _(Your Name)_`
 
 ## 0.2.1 - UNRELEASED
 
-- _()_
+- CI: Story-book build and stashed in Jenkins pipeline PR/Master/Release #438 _(4ydan)_
```

Comment:
please use a random line (see text above), then the rebase which is needed now could've been avoided.